### PR TITLE
docs(cli): complete missing operation descriptions and merge duplicate sections

### LINKED ISSUE #938: Production Readiness: Verify gossip ports are exposed via either a direct LoadBalancer or via HAProxy w/ TCP passthrough

### PR DESCRIPTION
**Pull request by deshmukhpranali**
## Description

This pull request changes the following:

This pull request updates the presentation_layer_cli_architecture.md file to complete missing descriptions and refactor similar content areas for better clarity and maintainability.

This update includes:
✅ Added missing descriptions for CLI operations across the following sections:

Consensus: Network, Node, State, Diagnostic

Deployment: Cluster, Config, State

Explorer, Mirror, Relay: Node operations

Quick Start: EVM, Single, Multi

Ledger: System, Account, Crypto

Keys: Consensus

### Related Issues

* Closes #938
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[x] This PR updated documentation
* \[x] This PR added no TODOs or commented out code
* \[x] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[x] All changes in this PR are included in the description
* \[x] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[x] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* Verified formatting, structure, and link integrity within the markdown preview.

The following was not tested:

* N/A (documentation-only PR)

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: docs/design/architecture/system/presentation_layer_cli_architecture.md [hunk 1]
```diff
@@ -538,80 +539,23 @@ operations associated with each resource.
...
-|----------------|----------------|-------------|
-| **Info**       | `info`         |             |
-| **Deploy**     | `deploy`       |             |
-| **Destroy**    | `destroy`      |             |
-
-<p align="right">
-:arrow_up_small: <a href="#table-of-contents">Back to top</a>
-</p>
-
-#### Multi
```
Comment (JeffreyDallas) [inline docs/design/architecture/system/presentation_layer_cli_architecture.md:608]: above lines should not be removed. they are placeholders.

File: docs/design/architecture/system/presentation_layer_cli_architecture.md [hunk 2]
```diff
@@ -538,80 +539,23 @@ operations associated with each resource.
...
-#### Node
-
-| Operation Name | Command Syntax | Description |
-|----------------|----------------|-------------|
-| **List**       | `list`         |             |
-| **Info**       | `info`         |             |
-| **Logs**       | `logs`         |             |
-| **Add**        | `add`          |             |
-| **Upgrade**    | `upgrade`      |             |
-| **Destroy**    | `destroy`      |             |
```
[the remark this answers is not in the document]
Comment (deshmukhpranali) [reply docs/design/architecture/system/presentation_layer_cli_architecture.md:576]: Ok, removing the second commit where I have removed all of these sections.

**Review by jeromy-cannon — APPROVED** [review]

**Review by alex-kuzmin-hg — APPROVED** [review]
LGTM
